script.js

```diff
@@ -32,3 +32,15 @@ let instructor = {
 instructor.facialhair = "got some"
 
 console.log(facialhair)
+
+$("#red").click(function()){
+  $("#red").toggleClass("active");
+}
+
+$("#yellow").click(function()){
+  $("#yellow").toggleClass("active");
+}
+
+$("#green").click(function()){
+  $("yellow").toggleClass("active");
+}
```

Comment:
👍 good work!

Keep in mind it can be helpful to target all of the elements that we care about initially:
```
var redLight = $("#red");
var yellowLight = $("#yellow");
var greenLight = $("#green");
...
```
that way we can just refer to them by name going forward